npm/js-cookie/2.2.0/package.json

```diff
@@ -4,6 +4,7 @@
 	"n4js": {
 		"projectType": "definition",
 		"definesPackage": "js-cookie",
+		"MainModule": "src/js-cookie",
```

Comment:
why isn't `src` the source container?

Reply:
don't know. I checked with the old manifest.n4mf file.